src/main/scala/midas/passes/package.scala

```diff
@@ -0,0 +1,156 @@
+package midas
+
+import firrtl._
+import firrtl.ir._
+import firrtl.passes.Pass
+import firrtl.Mappers._
+import firrtl.WrappedExpression._
+
+import annotations._
+import TargetToken.{Instance, OfModule}
+import firrtl.Utils.BoolType
+import firrtl.transforms.BlackBoxInlineAnno
+
+import scala.language.implicitConversions
+
+
```

Comment:
There's a bunch of great sugar in this Package object, it would be nice to see a little bit of Scala doc on it.

Reply:
Added